<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CsvAuto</class>
 <widget class="QMainWindow" name="CsvAuto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #171717;</string>
  </property>
  <widget class="QWidget" name="CsvAutoData">
   <widget class="QLabel" name="output_screen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>450</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="twod_plot">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>490</y>
      <width>285</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #171717;
color: white;
border-radius: 5px;
}
QPushButton:hover {
    background-color: #fd8a44; 
    border-color: #fd8a44; 
}</string>
    </property>
    <property name="text">
     <string>2D Plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>980</width>
      <height>130</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #212121;
color: white;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="upload_data">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>285</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #171717;
color: white;
border-radius: 5px;
}
QPushButton:hover {
    background-color: #fd8a44; 
    border-color: #fd8a44; 
}</string>
    </property>
    <property name="text">
     <string>Upload Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="threed_plot">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>490</y>
      <width>285</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #171717;
color: white;
border-radius: 5px;
}
QPushButton:hover {
    background-color: #fd8a44; 
    border-color: #fd8a44; 
}</string>
    </property>
    <property name="text">
     <string>3D Plot</string>
    </property>
   </widget>
   <zorder>output_screen</zorder>
   <zorder>label</zorder>
   <zorder>twod_plot</zorder>
   <zorder>upload_data</zorder>
   <zorder>threed_plot</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
